e2e/support/commands/downloads/downloadUtils-untyped.js

```diff
@@ -75,18 +75,20 @@ const verifyDownload = (fileName, options) => {
       result = cy
         .task("findFiles", { path: downloadsFolder, fileName })
         .then((files) => {
-          if (files !== null) {
-            if (files.length > 1) {
-              cy.log(
-                `**WARNING!** More than one file found for the **'${fileName}'** pattern: [${files}] - the first one **[${files[0]}]** will be used`,
-              );
-            }
-
-            return cy.task(
-              "isFileExist",
-              path.join(downloadsFolder, files[0] ?? ""),
+          if (!files || files.length === 0) {
+            return false;
```

Comment:
Before it worked wrong in case of a missing file, I had `truthy` check for non existing file name passed